### PR TITLE
Fixes #7358: Preliminary support for freebsd

### PR DESCRIPTION
**Pull request by peckpeck**
https://www.rudder-project.org/redmine/issues/7358

### CODE REVIEW

**Comment by jooooooon** [comment]
This looks good to me, as changes. However 1) it needs rebasing :), 2) I think the dependency updates should be in a separate ticket to make sure they show up in the changelog.

**Comment by peckpeck** [comment]
PR updated

**Comment by peckpeck** [comment]
PR updated

**Comment by peckpeck** [comment]
PR updated

**Comment by Kegeruneku** [comment]
Looks good !

File: rudder-agent/SOURCES/Makefile [hunk 1]
```diff
@@ -210,15 +215,9 @@ endif
 lmdb-source:
 ifeq ($(shell ../../build-caching get --force-config ./lmdb-source/ name=lmdb-src version=$(LMDB_RELEASE) >/dev/null 2>&1 || echo KO), KO)
 	# Original URL: http://ftp.fr.debian.org/debian/pool/main/l/lmdb/lmdb_$(LMDB_RELEASE).orig.tar.xz
-<<<<<<< HEAD
-	$(GET) $(TMP_DIR)/lmdb.tgz http://www.normation.com/tarball/lmdb-$(LMDB_RELEASE).tar.gz
-	gunzip < $(TMP_DIR)/lmdb.tgz | $(TAR) xf -
-	mv ./lmdb-$(LMDB_RELEASE) ./lmdb-source
-=======
 	$(WGET) -O $(TMP_DIR)/lmdb.tgz http://www.normation.com/tarball/lmdb/lmdb-$(LMDB_RELEASE).tar.gz
```
Comment (jooooooon) [inline rudder-agent/SOURCES/Makefile]: Pretty sure this should be ```$(GET)``` not ```$(WGET)```...

File: rudder-agent/SOURCES/Makefile [hunk 2]
```diff
@@ -210,15 +215,9 @@ endif
...
 	tar xzf $(TMP_DIR)/lmdb.tgz -C $(TMP_DIR)
 	mv $(TMP_DIR)/lmdb-$(LMDB_RELEASE) ./lmdb-source
```
Comment (jooooooon) [inline rudder-agent/SOURCES/Makefile]: Same here, should be gunzup, not tar xvz. The merge was resolved the wrong way round.

**Pull request merged**